recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "2022c" %}
+{% set version = "2022e" %}
 
 package:
   name: tzdata
   version: {{ version }}
 
 source:
   - url: https://data.iana.org/time-zones/releases/tzdata{{ version }}.tar.gz
-    sha256: 6974f4e348bf2323274b56dff9e7500247e3159eaa4b485dfa0cd66e75c14bfe
+    sha256: 8de4c2686dce3d1aae9030719e6814931c216a2d5e891ec3d332e6f6516aeccd
   - url: https://data.iana.org/time-zones/releases/tzcode{{ version }}.tar.gz
-    sha256: 3e7ce1f3620cc0481907c7e074d69910793285bffe0ca331ef1a6d1ae3ea90cc
+    sha256: d40280253980e89168e6be4275a852bf9521524d47684de3135b9a5ca387710b
 
 build:
   number: 0
+  noarch: generic
   # This package needs to be noarch because it uses POSIX system APIs.
   # There is no Windows port available and porting to Windows is not trivial.
+  # The Windows build will be skipped when built.
```

Comment:
I would add "... to avoid Prefect failures."

I would also move that line above "There is no Windows port..."